velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -713,6 +749,33 @@ MapVectorPtr VectorFuzzer::fuzzMap(
       values);
 }
 
+FlatMapVectorPtr VectorFuzzer::fuzzFlatMap(
+    const TypePtr& keyType,
+    const TypePtr& valueType,
+    vector_size_t size) {
+  // generate a vector of keys and extract the distinct ones
+  auto allKeys = fuzzFlatNotNull(keyType, getElementsVectorLength(opts_, size));
```

Comment:
we could also make this configurable (`opt_.flatMapMaxDistinctKeys`) or something similar.

Reply:
maybe we could create a `fuzzUniqueFlatNotNull()` function?